common/app/common/pages/ImageData.java

```diff
@@ -0,0 +1,36 @@
+package common.pages;
+
+import io.sphere.sdk.models.Base;
+import io.sphere.sdk.models.Image;
+
+public class ImageData extends Base {
+
+    private final Image image;
+
+    private ImageData(final Image image) {
+        this.image = image;
+    }
+
+    public static ImageData of(final Image image) {
+        return new ImageData(image);
+    }
+
+    public String getThumbImage() {
+        return nullIfEmpty(image.getUrl());
+    }
+
+    public String getBigImage() {
+        return nullIfEmpty(image.getUrl());
```

Comment:
It shouldn't be behaving like that since 2014. According to this test still works as it should:

https://github.com/jknack/handlebars.java/blob/master/handlebars/src/test/java/com/github/jknack/handlebars/Issue109.java#L21

We can check it together.